src/test/scala/io/iohk/ethereum/ledger/BranchResolutionSpec.scala

```diff
@@ -78,6 +77,7 @@ class BranchResolutionSpec
     "correctly handle a branch that goes up to the genesis block" in new BranchResolutionTestSetup {
       val headers = genesisHeader :: getChainHeadersNel(1, 10, genesisHeader.hash)
 
+      setHeaderInChain(genesisHeader.parentHash, result = false)
```

Comment:
the genesis block has a parent hash? O.o

Reply:
it doesn't, when trying to resolve the branch, it goes up until the genesis header and returns false, meaning it reached the genesis block. The genesis block is just a .copy(number = 0) of a valid block